Sup 34877 express capture bug while using jaws

salahmajadly opened the pull request:
### PR Info
- [x] Please functional test this PR
- [ ] Already tested in KMS and stand alone

#### Main changes
#### Known effected areas
#### Special considerations

odedm-kaltura reviewed (commented):
there are 4 issues in the ticket and fixes for 2 in the PR. what about the rest?

salahmajadly commented:
@odedm-kaltura 
for #1, fixed

for #2, I **cant reproduce** - maybe they are using an older version of JAWS - on 2023 I cant replicate

for #3, fixed 

for #4, its already been fixed by player (**can’t reproduce**)
